pyproject.toml

```diff
@@ -20,6 +20,11 @@ flask-sqlalchemy = "^3.1.1"
 faker = "^25.0.0"
 flask-login = "^0.6.3"
 
+[tool.ruff.pydocstyle]
+convention = "google"
```

Comment:
Add requirement for `google` style documentation: https://docs.astral.sh/ruff/faq/#does-ruff-support-numpy-or-google-style-docstrings